test/test_dispatch.cpp

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 
+#include "sparrow/layout/array_helper.hpp"
```

Comment:
Is it used ?

Reply:
Yes, the `array_element` function is declared in `array_helper.hpp` now.